server/helpers/validation.js

```diff
@@ -0,0 +1,12 @@
+function isNumberValid(number){
+    const arrNumber = number.split("");
+    arrNumber.splice(3,1);
+    return( number.length===11 &&
+            number.slice(3,4)==='-' && 
+            arrNumber.every(digit => digit>=0&&digit<=9) &&
```

Comment:
if its a digit it cant be  out of those bounds

Reply:
didn't understand.. digit are usuly between 0-9